.gitignore

```diff
@@ -12,8 +12,6 @@ capybara-*.html
 *.orig
 rerun.txt
 pickle-email-*.html
-# libraries should not add restrictions on the exact version of their dependencies to their hosting application
-Gemfile.lock
```

Comment:
I think this is incorrect. From a post from Yehuda Katz I started to remove this file from gems but in practice I think that doing this is bad because anyway it does not force anything on the code that uses the gems as they depend on the gemspec not the Gemfile* files.
On the other side it is good to define the exact versions of the libraries that were being used when the gem was implemented, so when you return to the gem you know what was exactly being used.

Reply:
I think this is incorrect too because I remove it from here and add gemfile.lock again to git... I don't understand the comment..